queries/pascal/locals.scm

```diff
@@ -0,0 +1,25 @@
+
+(root)                                   @local.scope
```

Comment:
these don't need to be prefixed with `@local`, it's just `@scope`/`@definition`/`@reference`.